### PR TITLE
Look for a Browserslist config if `browsers` are not set

### LINKED ISSUE #82: Read external Browserslist config if `browsers` is not set
If the `browsers` property is not set, we should check if there is an external Browserslist config in the repo (`package.json#browserslist`, ` .browserslistrc`, [etc.](https://github.com/browserslist/browserslist#queries)). You can call `browserslist()` with no arguments to get the value from a config: https://github.com/browserslist/browserslist#js-api

P.S. I can help to implement it

### PR DESCRIPTION
**Pull request by kytta**
Closes #82

### CODE REVIEW

**Comment by kytta** [comment]
I'd love to add a test for this, but I'm not sure how to temporarily modify the global environment to include an external Browserslist config

File: src/options.js [hunk 1]
```diff
@@ -11,11 +12,19 @@ function prepareGlobalOptions (pluginOptions = {}) {
     ...(pluginOptions.lightningcssOptions || {})
   };
 
+  let browserslistConfig;
+  if (pluginOptions.browsers != null) {
+    browserslistConfig = pluginOptions.browsers;
+  } else {
+    // try to find a browserslist config
+    browserslistConfig = browserslist.loadConfig({ path: path.resolve('.') });
```
Comment (onigoetz) [inline src/options.js:20]: You can use process.cwd() instead of path.resolve()

**Comment by onigoetz** [comment]
I think it can be done by putting files in a folder, including a .browserslistrc and pointing the test to this folder

**Review by onigoetz — COMMENTED** [review]
Thanks for the PR, I made a small comment that can be adjusted and added a comment on how to make a test

**Comment by kytta** [comment]
> I think it can be done by putting files in a folder, including a .browserslistrc and pointing the test to this folder

Will do this tomorrow 👌🏻

**Comment by onigoetz** [comment]
Hi @kytta did you have time to look into this ?

In can also take over to make this land, just tell me :)

**Comment by onigoetz** [comment]
Test added.
Thanks for the PR :D

**Comment by kytta** [comment]
Oh no, I'm so sorry I forgot about this PR 🙈 Thank you for finishing it up for me!

**Comment by onigoetz** [comment]
No worries, I also took a while to come back to it :)

**Pull request closed without merging**